app/src/main/java/org/gdg/frisbee/android/app/App.java

```diff
@@ -150,11 +153,17 @@ public void onCreate() {
         PrefUtils.increaseAppStartCount(this);
 
         // Initialize Picasso
+        // When we clone mOkHttpClient, it will use all the same cache and everything.
+        // Only the interceptors will be different.
+        // We shouldn't have the below interceptor in other instances.
+        OkHttpClient picassoClient = mOkHttpClient.clone();

```

Comment:
May I ask why you decides to clone the OkhttpClient rather than just use the current one?